tf_agents/environments/atari_preprocessing.py

```diff
@@ -31,13 +31,14 @@
 from __future__ import print_function
 
 import gin
+from gym.core import Wrapper
 from gym.spaces.box import Box
 import numpy as np
-import cv2
+import tensorflow as tf
```

Comment:
Not sure if it is better to depend on TF than on cv2 here?

Reply:
Sure, I'll send a separate PR. The motivation for this is that cv2 is only used here, and only to resize images, and since TF's resize function does the exact same thing, it feels like an unnecessary dependency (the one benefit is that cv2's implementation seems to be faster, at least on my machine).

Reply:
I reverted the cv2 change, I'll post a separate PR